CODING_STYLE.md

```diff
@@ -0,0 +1,199 @@
+# C++ Ethereum Coding Style
+
+## Code Formatting
+
+Use clang-format tool to format your changes, see [CONTRIBUTING](CONTRIBUTING.md) for details.
+
+
+## Namespaces
+
+a. No "using namespace" declarations in header files.
```

Comment:
This a-b-c-list is not rendered correctly by GitHub, use 1-2-3-list

Reply:
I missed this one.